README.md

```diff
@@ -16,7 +16,49 @@ http://127.0.0.1:1234/
 
 Try also
 http://127.0.0.1:1234/search
+
+# Other information:
+The content are stored on a google spreadsheet. You need permission to edit: 
+https://docs.google.com/spreadsheets/d/1QHKa3vUpht7zRl_LEzl3BlUbolz3ZiL8yKHzdBL42dY/edit
+
+# Deployment
+
+## Push to deploy
+The website is published on the domain name url.coderbunker.com
+
+The deployment is automatically triggered when something is pushed on the [deployment branch](https://github.com/coderbunker/inventory-server/tree/deployment).
+
+The deployment uses pm2 as suggested in [this tutorial](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-16-04).
```

Comment:
starting from this line, the following does not talk about deployment.
- server provisioning
- argumenting server provisioning
- codeship tutorial

Make a clear file PROVISIONING.md that focus only on the topic of setting up server ( including initial git clone and codeship set up)

All we are interested in README is how to deploy my last commit/merge to production.

Reply:
I can do that, but if I push it on this branch it will trigger an update on server side. 
Shall we merge it as is to master, and then I will commit these README changes to master